brain-marks/Settings/Views/AppIconListView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  AppIconListView.swift
+//  brain-marks
+//
+//  Created by Koty Stannard on 10/6/22.
+//
+
+import SwiftUI
+
+struct AppIconListView: View {
+    
+    @EnvironmentObject var appIconSettings: AppIconSettings
```

Comment:
Why use an `@EnvironmentObject` as opposed to passing it in the initializer and using `@ObservedObject` instead?